prov/rxm/src/rxm_cq.c

```diff
@@ -974,10 +974,15 @@ static inline ssize_t rxm_handle_atomic_resp(struct rxm_ep *rxm_ep,
 					   ofi_tx_cq_flags(tx_buf->pkt.hdr.op),
 					   tx_buf->app_context, tx_buf->flags);
 done:
-	if (tx_buf->pkt.hdr.atomic.op == ofi_op_atomic)
+	if (tx_buf->pkt.hdr.op == ofi_op_atomic) {
 		ofi_ep_wr_cntr_inc(&rxm_ep->util_ep);
-	else
+	} else if (tx_buf->pkt.hdr.op == ofi_op_atomic_compare ||
+		   tx_buf->pkt.hdr.op == ofi_op_atomic_fetch) {
 		ofi_ep_rd_cntr_inc(&rxm_ep->util_ep);
```

Comment:
@shefty should compare and fetch operations increment both read and write counters?

Reply:
No - I recently updated the man page to clarify this:
```
FI_READ
Increments the specified counter whenever an RMA read, atomic fetch, or atomic compare operation initiated from the endpoint has completed successfully or in error.
FI_WRITE
Increments the specified counter whenever an RMA write or base atomic operation initiated from the endpoint has completed successfully or in error.
FI_REMOTE_READ
Increments the specified counter whenever an RMA read, atomic fetch, or atomic compare operation is initiated from a remote endpoint that targets the given endpoint. Use of this flag requires that the endpoint be created using FI_RMA_EVENT.
FI_REMOTE_WRITE
Increments the specified counter whenever an RMA write or base atomic operation is initiated from a remote endpoint that targets the given endpoint. Use of this flag requires that the endpoint be created using FI_RMA_EVENT.
```
Only one counter is updated -- which would be a read counter in this case.